redhat-operators/bundles/amq-streams/bundle.1.0.0.yaml

```diff
@@ -0,0 +1,3177 @@
+---
```

Comment:
I was expecting to see the diffs with the old bundle file. Not sure why it shows the entire content of the bundle as new.

Reply:
I don't think we uploaded the old bundle, so this is the first bundle we've uploaded for amq

Reply:
We can `Install` `amq-streams` from Marketplace, so did we push it to quay directly?

Reply:
I pushed the previous bundle to quay. From now on, we'll only update bundles once the bundle PR is merged.